finetune/input_pipeline.py

```diff
@@ -372,6 +376,10 @@ def _text_to_ids(self, Xs, pad_token=None):
                             start_token, end_token = field_starts_and_ends[field]
                             if fv[0] != start_token:
                                 fv = np.concatenate(([start_token], fv))
+                                # Update start and end only once
+                                if field == EncodedOutput._fields[0]:
```

Comment:
This seems to imply that useful_start and useful_end were wrong before because they were never updated?